src/main/java/src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,36 @@
+package src.main.java.duke;
+
+/**
+ * Class to represent a src.main.java.duke.Todo task
+ */
+
+
+public class Todo extends Task {
+
+    private String task;
+    private boolean isDone;
+
+    Todo(String T, boolean D) {
+        super(T, D);
+        task = T;
+        isDone = D;
+    }
+
+    /**
+     * overridden method to mark the task as done
+     */
+    @Override
+    void markAsDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * overridden method to give the String representation of the task
+     *
+     * @return
```

Comment:
Did you forget to include the JavaDocs for the return statement?
I noticed the same issue in multiple places